user/installing-dependencies.md

```diff
@@ -254,20 +254,82 @@ addons:
 
 ## Installing Packages on OS X
 
-To install packages that are not included in the [default OS X environment](/user/reference/osx/#Compilers-and-Build-toolchain) use [Homebrew](http://brew.sh) in your `.travis.yml`. For example, to install beanstalk:
+To install packages that are not included in the [default OS X environment](/user/reference/osx/#Compilers-and-Build-toolchain) use [Homebrew](http://brew.sh) and our Homebrew addon in your `.travis.yml`. For example, to install beanstalk:
 
 ```yaml
-before_install:
-  - brew update # see note below
-  - brew install beanstalk
+addons:
+  homebrew:
+    packages:
+    - beanstalk
+```
+{: data-file=".travis.yml"}
+
+By default, the Homebrew addon will not run `brew update` before installing packages. `brew update` can take a long time and slow down your builds. If you need more up-to-date versions of packages than the snapshot on the build VM has, you can add `update: true` to the addon configuration:
+
+```yaml
+addons:
+  homebrew:
+    packages:
+    - beanstalk
+    update: true
+```
+{: data-file=".travis.yml"}
+
+### Taps and Casks
+
+The Homebrew addon supports more than just packages. It also supports installing [casks][homebrew-cask]: GUI apps and other things that need to run installers rather than being built from source code. Many casks are not very useful for automated building and testing, but there may be some that you need for your builds. You can add them to the `casks` key in the Homebrew addon configuration to install them:
```

Comment:
A few things on this paragraph:

1. "supports more than just packages" seems a bit wordy here. Just say "The addon also supports…".
1. "Many casks are not very useful for automated building and testing…" is probably unnecessary. Don't pass judgement on what is useful or not for the users' builds. If they are adding casks, they need it.
1. Also, if the user is adding a cask, it seems acceptable to assume that they know what a cask is.

Reply:
Ok, I can adjust this.